tests/e2e-instrumentation/instrumentation-java-multicontainer/chainsaw-test.yaml

```diff
@@ -49,3 +49,37 @@ spec:
     catch:
       - podLogs:
           selector: app=my-java-multi
+  - name: "wait for app to be ready"
+    try:
+      - sleep:
+          duration: 5s
+  - name: Make a request to the app
+    try:
+      - script:
+          content: |
+            #!/bin/bash
+            # set -ex
+            pod=$(kubectl get pods -n $NAMESPACE | awk '{print $1}' | tail -n 1)
+            kubectl get --raw /api/v1/namespaces/$NAMESPACE/pods/${pod}:8080/proxy/
```

Comment:
this is new to me :) 

@IshwarKanse does it work on OCP?

Reply:
Looking at this, I really wonder if we shouldn't use e2e_framework and/or terratest for these tests. The main problem is really getting data out of the cluster, as we don't have a remote we can easily query. I can think of a few solutions to do this in kind, but they won't work on OpenShift. It'd be best if these could simply be Go tests we could run on a host against any cluster.

Reply:
This can be implemented natively using Chainsaw 
and avoid using the script. https://github.com/kyverno/chainsaw/blob/main/testdata/e2e/examples/proxy/chainsaw-test.yaml 

We were also discussing on implementing some kind of custom store in Chainsaw which gathers all the signals and would be useful for asserting. https://docs.google.com/document/d/1G4uDMrC3okF2q2qyNYTg2qj7yxNBjSTGgn8pBEXEeiw/edit#heading=h.8mm7ay5rgjs6 

@eddycharly Do we have any decision on what custom store we can use for testing observability data. ?